news/2017-08-17-happystick-summer-tour-concludes.md

```diff
@@ -0,0 +1,50 @@
+After an arduous and exciting five weeks of matches in which more than 700 initial contestants were narrowed down to a mere 26, the HappyStick Summer Tour reached its climax Sunday, August 13 with the Grand Final matches. Over the course of the tournament, 768 players from [x] countries across 13 rank brackets ranging from the 250 - 500 to 100,000+ tiers fought their way through a grand total of 537 matches from Group Stages through Bracket Stages until only two players remained in each. Those select few fought it out mano a mano on the grand stage of the finals where they clashed for the crown of Summer Tour Champion.
+
+![](/wiki/shared/2017-08-17-happystick-summer-tour-concludes/banner.jpg)
+
+As the dust settled on the Winners and Losers Finals of the double-elimination tournament, only one of the 26 Grand Finalists was a repeat from the Spring Tour, leaving us in the thrilling position of witnessing all-new champions across the board. And thrilled we were, as seven of the 13 matches would go to a bracket reset as the players fighting their way from the Losers Finals sought to show that they deserved to be there just as much as those who hadn’t dropped a match and advanced from the Winners Brackets. Would the favorites find victory, or would the underdogs claw their way through the grueling task of winning back to back best of 11-map series to find glory? Read on for results and VODs of our featured matches!
```

Comment:
i also shunted this below the banner - the banner **always** comes after the entry paragraph